deploy/service.yaml

```diff
@@ -51,7 +51,7 @@ spec:
       serviceAccountName: keptn-dynatrace-sli-service
       containers:
         - name: dynatrace-sli-service
-          image: keptncontrib/dynatrace-sli-service:0.8.0
+          image: keptncontrib/dynatrace-sli-service:0.9.0
```

Comment:
Should we use `distributor:0.8.1` ?

Reply:
clarified with @bacherfl - not neccessary